real,user,sys
/usr/bin/env: th: No such file or directory
Traceback (most recent call last):
  File "../demos/compare.py", line 101, in <module>
    d = getRep(img1) - getRep(img2)
  File "../demos/compare.py", line 92, in getRep
    rep = net.forward(alignedFace)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 156, in forward
    rep = self.forwardPath(t)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 113, in forwardPath
    """.format(self.cmd, self.p.stdout.read()))
Exception: 


OpenFace: `openface_server.lua` subprocess has died.

+ Is the Torch command `th` on your PATH? Check with `which th`.

+ If `th` is on your PATH, try running `./util/profile-network.lua`
  to see if Torch can correctly load and run the network.

  + If this gives illegal instruction errors, see the section on
    this in our FAQ at http://cmusatyalab.github.io/openface/faq/

  + In Docker, use a Bash login shell or source
     /root/torch/install/bin/torch-activate for the Torch environment.

+ See this GitHub issue if you are running on a non-64-bit machine:
  https://github.com/cmusatyalab/openface/issues/42

+ Please post further issues to our mailing list at
  https://groups.google.com/forum/#!forum/cmu-openface

Diagnostic information:

cmd: ['/usr/bin/env', 'th', '/usr/local/lib/python2.7/dist-packages/openface/openface_server.lua', '-model', '/root/openface/demos/../models/openface/nn4.small2.v1.t7', '-imgDim', '96']

============

stdout: 

2.372,2.270,0.090
/usr/bin/env: th: No such file or directory
Traceback (most recent call last):
  File "../demos/compare.py", line 101, in <module>
    d = getRep(img1) - getRep(img2)
  File "../demos/compare.py", line 92, in getRep
    rep = net.forward(alignedFace)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 156, in forward
    rep = self.forwardPath(t)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 113, in forwardPath
    """.format(self.cmd, self.p.stdout.read()))
Exception: 


OpenFace: `openface_server.lua` subprocess has died.

+ Is the Torch command `th` on your PATH? Check with `which th`.

+ If `th` is on your PATH, try running `./util/profile-network.lua`
  to see if Torch can correctly load and run the network.

  + If this gives illegal instruction errors, see the section on
    this in our FAQ at http://cmusatyalab.github.io/openface/faq/

  + In Docker, use a Bash login shell or source
     /root/torch/install/bin/torch-activate for the Torch environment.

+ See this GitHub issue if you are running on a non-64-bit machine:
  https://github.com/cmusatyalab/openface/issues/42

+ Please post further issues to our mailing list at
  https://groups.google.com/forum/#!forum/cmu-openface

Diagnostic information:

cmd: ['/usr/bin/env', 'th', '/usr/local/lib/python2.7/dist-packages/openface/openface_server.lua', '-model', '/root/openface/demos/../models/openface/nn4.small2.v1.t7', '-imgDim', '96']

============

stdout: 

2.390,2.300,0.080
/usr/bin/env: th: No such file or directory
Traceback (most recent call last):
  File "../demos/compare.py", line 101, in <module>
    d = getRep(img1) - getRep(img2)
  File "../demos/compare.py", line 92, in getRep
    rep = net.forward(alignedFace)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 156, in forward
    rep = self.forwardPath(t)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 113, in forwardPath
    """.format(self.cmd, self.p.stdout.read()))
Exception: 


OpenFace: `openface_server.lua` subprocess has died.

+ Is the Torch command `th` on your PATH? Check with `which th`.

+ If `th` is on your PATH, try running `./util/profile-network.lua`
  to see if Torch can correctly load and run the network.

  + If this gives illegal instruction errors, see the section on
    this in our FAQ at http://cmusatyalab.github.io/openface/faq/

  + In Docker, use a Bash login shell or source
     /root/torch/install/bin/torch-activate for the Torch environment.

+ See this GitHub issue if you are running on a non-64-bit machine:
  https://github.com/cmusatyalab/openface/issues/42

+ Please post further issues to our mailing list at
  https://groups.google.com/forum/#!forum/cmu-openface

Diagnostic information:

cmd: ['/usr/bin/env', 'th', '/usr/local/lib/python2.7/dist-packages/openface/openface_server.lua', '-model', '/root/openface/demos/../models/openface/nn4.small2.v1.t7', '-imgDim', '96']

============

stdout: 

2.343,2.270,0.060
/usr/bin/env: th: No such file or directory
Traceback (most recent call last):
  File "../demos/compare.py", line 101, in <module>
    d = getRep(img1) - getRep(img2)
  File "../demos/compare.py", line 92, in getRep
    rep = net.forward(alignedFace)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 156, in forward
    rep = self.forwardPath(t)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 113, in forwardPath
    """.format(self.cmd, self.p.stdout.read()))
Exception: 


OpenFace: `openface_server.lua` subprocess has died.

+ Is the Torch command `th` on your PATH? Check with `which th`.

+ If `th` is on your PATH, try running `./util/profile-network.lua`
  to see if Torch can correctly load and run the network.

  + If this gives illegal instruction errors, see the section on
    this in our FAQ at http://cmusatyalab.github.io/openface/faq/

  + In Docker, use a Bash login shell or source
     /root/torch/install/bin/torch-activate for the Torch environment.

+ See this GitHub issue if you are running on a non-64-bit machine:
  https://github.com/cmusatyalab/openface/issues/42

+ Please post further issues to our mailing list at
  https://groups.google.com/forum/#!forum/cmu-openface

Diagnostic information:

cmd: ['/usr/bin/env', 'th', '/usr/local/lib/python2.7/dist-packages/openface/openface_server.lua', '-model', '/root/openface/demos/../models/openface/nn4.small2.v1.t7', '-imgDim', '96']

============

stdout: 

2.343,2.260,0.070
/usr/bin/env: th: No such file or directory
Traceback (most recent call last):
  File "../demos/compare.py", line 101, in <module>
    d = getRep(img1) - getRep(img2)
  File "../demos/compare.py", line 92, in getRep
    rep = net.forward(alignedFace)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 156, in forward
    rep = self.forwardPath(t)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 113, in forwardPath
    """.format(self.cmd, self.p.stdout.read()))
Exception: 


OpenFace: `openface_server.lua` subprocess has died.

+ Is the Torch command `th` on your PATH? Check with `which th`.

+ If `th` is on your PATH, try running `./util/profile-network.lua`
  to see if Torch can correctly load and run the network.

  + If this gives illegal instruction errors, see the section on
    this in our FAQ at http://cmusatyalab.github.io/openface/faq/

  + In Docker, use a Bash login shell or source
     /root/torch/install/bin/torch-activate for the Torch environment.

+ See this GitHub issue if you are running on a non-64-bit machine:
  https://github.com/cmusatyalab/openface/issues/42

+ Please post further issues to our mailing list at
  https://groups.google.com/forum/#!forum/cmu-openface

Diagnostic information:

cmd: ['/usr/bin/env', 'th', '/usr/local/lib/python2.7/dist-packages/openface/openface_server.lua', '-model', '/root/openface/demos/../models/openface/nn4.small2.v1.t7', '-imgDim', '96']

============

stdout: 

2.325,2.240,0.070
/usr/bin/env: th: No such file or directory
Traceback (most recent call last):
  File "../demos/compare.py", line 101, in <module>
    d = getRep(img1) - getRep(img2)
  File "../demos/compare.py", line 92, in getRep
    rep = net.forward(alignedFace)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 156, in forward
    rep = self.forwardPath(t)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 113, in forwardPath
    """.format(self.cmd, self.p.stdout.read()))
Exception: 


OpenFace: `openface_server.lua` subprocess has died.

+ Is the Torch command `th` on your PATH? Check with `which th`.

+ If `th` is on your PATH, try running `./util/profile-network.lua`
  to see if Torch can correctly load and run the network.

  + If this gives illegal instruction errors, see the section on
    this in our FAQ at http://cmusatyalab.github.io/openface/faq/

  + In Docker, use a Bash login shell or source
     /root/torch/install/bin/torch-activate for the Torch environment.

+ See this GitHub issue if you are running on a non-64-bit machine:
  https://github.com/cmusatyalab/openface/issues/42

+ Please post further issues to our mailing list at
  https://groups.google.com/forum/#!forum/cmu-openface

Diagnostic information:

cmd: ['/usr/bin/env', 'th', '/usr/local/lib/python2.7/dist-packages/openface/openface_server.lua', '-model', '/root/openface/demos/../models/openface/nn4.small2.v1.t7', '-imgDim', '96']

============

stdout: 

2.343,2.240,0.090
/usr/bin/env: th: No such file or directory
Traceback (most recent call last):
  File "../demos/compare.py", line 101, in <module>
    d = getRep(img1) - getRep(img2)
  File "../demos/compare.py", line 92, in getRep
    rep = net.forward(alignedFace)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 156, in forward
    rep = self.forwardPath(t)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 113, in forwardPath
    """.format(self.cmd, self.p.stdout.read()))
Exception: 


OpenFace: `openface_server.lua` subprocess has died.

+ Is the Torch command `th` on your PATH? Check with `which th`.

+ If `th` is on your PATH, try running `./util/profile-network.lua`
  to see if Torch can correctly load and run the network.

  + If this gives illegal instruction errors, see the section on
    this in our FAQ at http://cmusatyalab.github.io/openface/faq/

  + In Docker, use a Bash login shell or source
     /root/torch/install/bin/torch-activate for the Torch environment.

+ See this GitHub issue if you are running on a non-64-bit machine:
  https://github.com/cmusatyalab/openface/issues/42

+ Please post further issues to our mailing list at
  https://groups.google.com/forum/#!forum/cmu-openface

Diagnostic information:

cmd: ['/usr/bin/env', 'th', '/usr/local/lib/python2.7/dist-packages/openface/openface_server.lua', '-model', '/root/openface/demos/../models/openface/nn4.small2.v1.t7', '-imgDim', '96']

============

stdout: 

2.354,2.270,0.070
/usr/bin/env: th: No such file or directory
Traceback (most recent call last):
  File "../demos/compare.py", line 101, in <module>
    d = getRep(img1) - getRep(img2)
  File "../demos/compare.py", line 92, in getRep
    rep = net.forward(alignedFace)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 156, in forward
    rep = self.forwardPath(t)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 113, in forwardPath
    """.format(self.cmd, self.p.stdout.read()))
Exception: 


OpenFace: `openface_server.lua` subprocess has died.

+ Is the Torch command `th` on your PATH? Check with `which th`.

+ If `th` is on your PATH, try running `./util/profile-network.lua`
  to see if Torch can correctly load and run the network.

  + If this gives illegal instruction errors, see the section on
    this in our FAQ at http://cmusatyalab.github.io/openface/faq/

  + In Docker, use a Bash login shell or source
     /root/torch/install/bin/torch-activate for the Torch environment.

+ See this GitHub issue if you are running on a non-64-bit machine:
  https://github.com/cmusatyalab/openface/issues/42

+ Please post further issues to our mailing list at
  https://groups.google.com/forum/#!forum/cmu-openface

Diagnostic information:

cmd: ['/usr/bin/env', 'th', '/usr/local/lib/python2.7/dist-packages/openface/openface_server.lua', '-model', '/root/openface/demos/../models/openface/nn4.small2.v1.t7', '-imgDim', '96']

============

stdout: 

2.345,2.300,0.030
/usr/bin/env: th: No such file or directory
Traceback (most recent call last):
  File "../demos/compare.py", line 101, in <module>
    d = getRep(img1) - getRep(img2)
  File "../demos/compare.py", line 92, in getRep
    rep = net.forward(alignedFace)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 156, in forward
    rep = self.forwardPath(t)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 113, in forwardPath
    """.format(self.cmd, self.p.stdout.read()))
Exception: 


OpenFace: `openface_server.lua` subprocess has died.

+ Is the Torch command `th` on your PATH? Check with `which th`.

+ If `th` is on your PATH, try running `./util/profile-network.lua`
  to see if Torch can correctly load and run the network.

  + If this gives illegal instruction errors, see the section on
    this in our FAQ at http://cmusatyalab.github.io/openface/faq/

  + In Docker, use a Bash login shell or source
     /root/torch/install/bin/torch-activate for the Torch environment.

+ See this GitHub issue if you are running on a non-64-bit machine:
  https://github.com/cmusatyalab/openface/issues/42

+ Please post further issues to our mailing list at
  https://groups.google.com/forum/#!forum/cmu-openface

Diagnostic information:

cmd: ['/usr/bin/env', 'th', '/usr/local/lib/python2.7/dist-packages/openface/openface_server.lua', '-model', '/root/openface/demos/../models/openface/nn4.small2.v1.t7', '-imgDim', '96']

============

stdout: 

2.370,2.340,0.020
/usr/bin/env: th: No such file or directory
Traceback (most recent call last):
  File "../demos/compare.py", line 101, in <module>
    d = getRep(img1) - getRep(img2)
  File "../demos/compare.py", line 92, in getRep
    rep = net.forward(alignedFace)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 156, in forward
    rep = self.forwardPath(t)
  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 113, in forwardPath
    """.format(self.cmd, self.p.stdout.read()))
Exception: 


OpenFace: `openface_server.lua` subprocess has died.

+ Is the Torch command `th` on your PATH? Check with `which th`.

+ If `th` is on your PATH, try running `./util/profile-network.lua`
  to see if Torch can correctly load and run the network.

  + If this gives illegal instruction errors, see the section on
    this in our FAQ at http://cmusatyalab.github.io/openface/faq/

  + In Docker, use a Bash login shell or source
     /root/torch/install/bin/torch-activate for the Torch environment.

+ See this GitHub issue if you are running on a non-64-bit machine:
  https://github.com/cmusatyalab/openface/issues/42

+ Please post further issues to our mailing list at
  https://groups.google.com/forum/#!forum/cmu-openface

Diagnostic information:

cmd: ['/usr/bin/env', 'th', '/usr/local/lib/python2.7/dist-packages/openface/openface_server.lua', '-model', '/root/openface/demos/../models/openface/nn4.small2.v1.t7', '-imgDim', '96']

============

stdout: 

2.372,2.230,0.130
